doc/sphinx/source/community/backward_compatibility.rst

```diff
@@ -0,0 +1,263 @@
+*backward-incompatible changes*” section below).
+We organize monthly community meetings where *recipe developers* can learn about
+the latest developments and everyone is welcome to join, ask questions, and
+provide feedback.
+
+To meet the needs of users and funders, we should take reproducibility
+of older results seriously, but this should not hold us back from
+advancing our tools. We can support this by uploading a well tested
+container image to an archive that provides a DOI and by providing clear
+instructions on how to use such containers.
+
+Helping developers to upgrade
+-----------------------------
+
+*Recipe users* of ESMValTool will be able to successfully run
+*integrated recipes* using a *release*, since all
+*backwards-incompatible changes* introduced between *releases* will have
+been fixed before the *release* is created. Please note the
+:ref:`broken recipe policy<broken-recipe-policy>`.
+
+However, *recipe developers* working on *user recipes* must be provided
+with information to enable them to adapt their code to resolve issues
+related to *backwards-incompatible changes* when *backwards-incompatible
+changes* are introduced to the main branch / when a *release* of
+ESMValTool is created.
+
+Guidance on handling *backward-incompatible changes*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As well as helping users to handle *backward-incompatible changes*, the
+policy and surrounding tools must help developers avoid making
+*backwards-incompatible changes*. Not many ideas developed on this yet,
+but components should include:
+
+-  Testing; *backwards-incompatible changes* should be discovered as
+   early in the development process as possible. This motivates
+   continued investment in automated testing.
+-  Guidance on how to minimise the likelihood of introducing
+   *backward-incompatible changes* and how to use deprecation warnings
+   when needed (see `developer
+   guidance <https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#backward-compatibility>`__).
+-  Instructions on how to provide the information that *recipe
+   developers* need to adapt to the *backward-incompatible change*,
+   e.g. the text that will go into *release* notes if we follow the
+   proposal above.
+-  General instructions for *recipe developers* working on *user
+   recipes* to enable them to adapt their code related to
+   *backwards-incompatible changes* (see `ESMValTool_Tutorial: issue
+   #263 <https://github.com/ESMValGroup/ESMValTool_Tutorial/issues/263>`__).
+-  The developer or reviewer must tag the core development team to
+   notify them of the *backward-incompatible change*, and give at least
+   2 weeks for objections to be raised before merging to the main
+   branch. If a strong objection is raised the backwards-incompatible
+   change should not be merged until the objection is resolved.
+
+During the *release* process, the following information must be
```

Comment:
This section needs to be linked from
- https://docs.esmvaltool.org/projects/esmvalcore/en/latest/contributing.html#add-release-notes
- https://docs.esmvaltool.org/en/latest/community/release_strategy.html#add-release-notes

Reply:
Added references in both of the above: https://github.com/ESMValGroup/ESMValCore/commit/8af608a679866a2ce479f942e223f872ef66ab08 and https://github.com/ESMValGroup/ESMValTool/commit/14b482df3dddeb0355cc2b78bb617b4b4193f2e5. The first one in particular should be reviewed (@bouweandela ?), as the release notes instructions previously said nothing about a section on backward incompatible changes, and assumed that the release notes will be automated. My change to the instructions introduces manual editing of the ESMValCore release notes.